CreatubblesAPIClient/Sources/Mapper/GalleryMapper.swift

```diff
@@ -77,6 +78,7 @@ class GalleryMapper: Mappable {
 
         challengePublishedAt <- (map["attributes.challenge_published_at"], APIClientDateTransform.sharedTransform)
         galleryInstructionRelationships <- map["relationships.gallery_howto_sections.data"]
+        connectedPartnersRelashionships <- map["relationships.connected_partner_applications.data"]
```

Comment:
Maybe it might make sense to call it `connectedPartnerApplicationsRelationships`? It might make things easier to connect later when reading new requirements referring to partner applications vs partners. Just a thought, but maybe it's OK as-is.